modules/special-rules/pages/cavalry.adoc

```diff
@@ -4,11 +4,33 @@ Despite the abundance and variety of technologies in the galaxy of the 41^st^ mi
 
 Of course, riders and their mounts vary enormously -- humans on horses, denizens of Chaos on daemonic steeds, orks on cyboars, and many more.
 
+Cavalry forgo some of the advantages of infantry -- they find it difficult to take cover, to move through buildings or rubble, and so on.
+But they make up for this with speed.
+
+[NOTE.e40k]
+====
+In comparison to _Epic 40,000_, we've separated the Armour bonus part of the _Cavalry_ ability into our new {armour-1} ability.
+====
+
 ---
 
-Apply these rules to {cavalry} units, relative to the base units:
+Where your unit has the {cavalry} ability, apply these changes relative to its base unit:
+
+* Create a split Speed value.
```

Comment:
A minor suggestion to cover a gap between creating the split value and applying it to the profile:

```suggestion
* Create and apply a split Speed value.
```

Reply:
Interesting. I'm not sure you can create it without also applying it, since to 'create' it you'd have to put it on the data sheet, so you've already applied it to the data sheet. :-D However, if you would argue this point and you think it reads better, I'm happy to go ahead with your suggestion here and on Jump Packs. Please feel free to apply them both in that case.

Though I suppose that now you've made me think of it, the thing that is really missing here is mention of the data sheet (it's implied by extension, but not explicit)... which could be in the bullet, or probably even better in the lead-in sentence. But then we should also make other similar things match. So, I'm happy to leave that for now too -- it's the type of thing that I think is better to refine across the whole section sometime in future.

Reply:
> Interesting. I'm not sure you can create it without also applying it, since to 'create' it you'd have to put it on the data sheet, so you've already applied it to the data sheet. :-D

That's exactly the problem - you've created a value, and then done nothing with the created value. Compare to other modifying ability texts like 'Add X to their Y value' or 'double their Y value', which inherently refer to the unit's data sheet.

In programming terms, you've created a variable and put a value in it, but that variable is 'invisible' because it's not referred to or used anywhere.

Hmm. Maybe like this?

```suggestion
* The unit has a split Speed value, created in this way.
```

Or

```suggestion
* The unit's Speed characteristic has two values.
```

Reply:
I think the second one is closer to existing Data sheet modifier abilities.

Reply:
>The unit has a split Speed value, created in this way.
>Or
>The unit's Speed characteristic has two values.

The first one still doesn't actually address the point -- the data sheet is still implicit. The second one does address the point because it references 'characteristic', which is on the data sheet.

However... the unit doesn't 'have' ('has') anything until you make it so. Both of these suggestions are in passive case -- so they don't fit with the active case, second person, imperative style of the surrounding lines and most of the rest of publication.

I could write a sentence in active case etc. that does reference the data sheet -- but as I mentioned, probably the best solution is to change the lead-in sentence (so that you don't have to repeat in each bullet where there are multiple). And then apply the same across all similar cases. So, could do it now, but I'm happy to leave it for the moment and fix in one swoop later, probably at the same time as other similar consistency stuff across the whole section.

As it is, I have only to review the transport topic and check it back with you and the whole PR is ready to merge.

Reply:
Or actually, I think this would do it with no bother, in similar fashion to 'Add X to their Y value'...
```suggestion
* Make its Speed value into a split Speed value.
```

Reply:
> Or actually, I think this would do it with no bother, in similar fashion to 'Add X to their Y value'...

Let's go with that. I'll copy it across to Jump Packs and commit them both.